velox/functions/prestosql/types/JsonCastOperator.cpp

```diff
@@ -702,7 +702,6 @@ struct CastFromJsonTypedImpl {
           auto size = unescapeSizeForJsonCast(json.data(), json.size());
           vectorWriter.resize(size);
           unescapeForJsonCast(json.data(), json.size(), vectorWriter.data());
-          vectorWriter.finalize();
```

Comment:
Where does commit get called? (In particular will this work for scalar strings not wrapped inside a complex value too?)